aspnetcore/blazor/components/prerender.md

```diff
@@ -197,85 +191,6 @@ In the server project's `Program` file, register the service:
 builder.Services.TryAddScoped<IWebAssemblyHostEnvironment, ServerHostEnvironment>();
 ```
 
-At this point, the <xref:Microsoft.AspNetCore.Components.WebAssembly.Hosting.IWebAssemblyHostEnvironment> service can be injected into an interactive WebAssembly or Auto component and used as shown in the [Read the environment in a Blazor WebAssembly app](xref:blazor/fundamentals/environments#read-the-environment-in-a-blazor-webassembly-app) section of the *Environments* article.
-
-The preceding example can demonstrate that it's possible to have a different server environment than client environment, which isn't recommended and may lead to arbitrary results. When setting the environment in a Blazor Web App, it's best to match server and `.Client` project environments. Consider the following scenario in a test app:
```

Comment:
I don't think the prerendering article is really the place to discuss the problems that might happen if you somehow set up a different environment on the client and the server. If we think this content is important to keep, maybe it should go in one of the environment docs?

Reply:
I'll move this bit over to the *Environments* article for the next commit and round of reviews there.